ext/mimemagic/Rakefile

```diff
@@ -3,7 +3,15 @@ require "rake/clean"
 
 def locate_mime_database
   possible_paths = [
-    (File.expand_path(ENV["FREEDESKTOP_MIME_TYPES_PATH"]) if ENV["FREEDESKTOP_MIME_TYPES_PATH"]),
+    (File.expand_path(ENV["FREEDESKTOP_MIME_TYPES_PATH"]) if ENV["FREEDESKTOP_MIME_TYPES_PATH"])
```

Comment:
[suggestion]
Should you consider creating a small method to called *listPossiblePaths* and avoid having these hard to read lines?

For example:

```ruby
def listPossibleMimePaths
  paths = []
  paths = File.expand_path(ENV["FREEDESKTOP_MIME_TYPES_PATH"] if ENV["FREEDESKTOP_MIME_TYPES_PATH"]
...

paths
end
```